WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -114,11 +150,13 @@
         </li>
       </ul>
 
-      <form class="navbar-form navbar-left" role="search">
-        <%= wca_omni_search %>
-      </form>
 
       <ul class="nav navbar-nav navbar-right">
+        <li class="search">
```

Comment:
One of the tests fails because it uses `.search` as the selector (on another page) and now it sees this element as well. The simplest solution to that would be for you to change this class to to something else (here and in the css). I think `search-container` would be a pretty accurate name =)